.github/workflows/test.yml

```diff
@@ -82,6 +82,19 @@ jobs:
         if: github.event_name != 'schedule'
         run: shellcheck -o all -S style -s sh $(find . -iname "*.sh")
 
+  musl-test:
```

Comment:
This job will need to either:

1. Be set as a required status check in the `phylum-dev/cli` settings. This can be done by someone with permissions to the repo and _may_ have to wait until after this PR is merged to `main`.
2. Included in the `test` "rollup job" below...which is already set as a required status check. Doing it that way would look something like this:

```yaml
  # This job reports the results of the test matrix above
  test:
    if: always()
    needs: [test-matrix, musl-test]
    runs-on: ubuntu-latest
    steps:
      - if: needs.test-matrix.result != 'success' || needs.musl-test.result != 'success'
        name: Fail the build
        run: exit 1
```